docs-chef-io/content/inspec/resources/groups.md

```diff
@@ -74,6 +74,20 @@ The `members` property tests the members that belong to a group:
     its('members') { should include 'root' }
     its('members') { should include 'Administrator' }
 
+where `members` returns
+- Array of group members for **Windows Platform**.
+Example: `["member1", "member2"]`
+- Single element array with CSV formatted string of group members for **Non-Windows Platforms**.
+Example: `["member1,member2"]`
```

Comment:
```suggestion
where `members` returns:

- an array of group members for **Windows Platform**.

   Example: `["member1", "member2"]`

- a CSV formatted string of group members for **Non-Windows Platforms**.

   Example: `"member1,member2"`
```

Reply:
@IanMadd Is there any reason behind the edit on line 83 and 85 because it goes against the fact that it returns a single element array of format `["member1,member2"]` ?
We should not let this information change I feel because it does return an array in groups resource on line 83 & 85.

Reply:
If I am not wrong we are introducing new property so that it returns the array of members regardless of any OS, so as per me we should return the result in the format of ["member1", "member2"] regardless of any OS and if they are different set of members and not the ["member1, member2"]..

Reply:
@Vasu1105 Yes this document change above is for already existing property `members` to clarify more on document that how it is already working for different platforms. For `members_array` property the documents always states to return an array irrespective of the platform.

Reply:
How about this instead:

`a single element array that contains a CSV string of group members for **Non-Windows Platforms**.`